pkg/resource/schemas.go

```diff
@@ -106,6 +122,17 @@ func (r *SchemaRepository) Init(providerName, providerVersion string, schema map
 	return nil
 }
 
+func (r SchemaRepository) SetFlags(typ string, flags ...Flags) {
```

Comment:
This is minor but you could have used the `resource.ResourceType` type here

Reply:
Oops sorry I clicked on merge by the same time ...